### PR TITLE
Further lining up with Core Plugwise

### PR DESCRIPTION
**Pull request by bouwew**
More lining up with Core Plugwise

### CODE REVIEW

File: scripts/core-testing.sh [hunk 1]
```diff
@@ -105,10 +105,10 @@ if [ -z "${GITHUB_ACTIONS}" ] || [ "$1" == "core_prep" ] ; then
 		fi
 		cd "${coredir}" || exit
 		echo ""
-		echo " ** Resetting to dev **"
+		echo " ** Resetting to master **"
```
Comment (CoMPaTech) [inline scripts/core-testing.sh:108]: While I do understand this caused a recent issue, we should align with dev, not master. It's better to wait a little with a release + also we won't know that something breaks this way up front and always will find out after the fact.

**Review by CoMPaTech — CHANGES_REQUESTED** [review]

File: scripts/core-testing.sh (hunk 1 above)
Comment (CoMPaTech) [reply scripts/core-testing.sh:108]: Added PR to for this PR-branch with suggestions

**Comment by CoMPaTech** [comment]
Please set labels :)

File: scripts/core-testing.sh (hunk 1 above)
Comment (bouwew) [reply scripts/core-testing.sh]: PR has been merged.

**Review by CoMPaTech — APPROVED** [review]

**Pull request merged**